src/Formula.purs

```diff
@@ -80,22 +88,39 @@ derive instance ordFormula :: Ord Formula
 
 -- | Shows formulas using least amount of parentheses wrt. precedence.
 instance showFormula :: Show Formula where
-  show = showPrec 1
+  show = showPrec symbol 1
     where
-    parens s = "(" <> s <> ")"
-
-    optParens b s = if b then parens s else s
+      symbol s = s
 
-    showPrec n = case _ of
-      Predicate p [] -> p
-      Predicate "=" [ x, y ] -> show x <> " = " <> show y
-      Predicate p args -> p <> parens (joinWith ", " (show <$> args))
-      Not f -> "¬" <> showPrec 4 f
-      Forall x f -> "∀" <> show x <> " " <> showPrec 4 f
-      Exists x f -> "∃" <> show x <> " " <> showPrec 4 f
-      And a b -> optParens (n > 3) $ showPrec 3 a <> " ∧ " <> showPrec 4 b
-      Or a b -> optParens (n > 2) $ showPrec 2 a <> " ∨ " <> showPrec 3 b
-      Implies a b -> optParens (n > 1) $ showPrec 2 a <> " → " <> showPrec 1 b
+instance latexFormula :: Latex Formula where
+  toLatex = showPrec symbol 1
```

Comment:
My first reaction was that a simple string search&replace would suffice, but this is fine too. (Maybe have a trailing space or `{}` after all TeX symbols for consistency?)

Reply:
Seems like the tests will need updating after this change. :)